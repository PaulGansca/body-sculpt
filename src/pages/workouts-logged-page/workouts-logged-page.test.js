import { shallow } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import WorkoutsLoggedPage from './workouts-logged-page';
import WorkoutsLoggedPageContainer from './workouts-logged-page.container';

let wrapper;

describe('<WorkoutsLoggedPage />', () => {
    it("container renders workout based on id", () => {
        const mockProps = {
            match: {params: {workoutId: 1}},
            isUserLoading: true,
            isWorkoutLoading: true,
            getState: jest.fn(),
            subscribe: jest.fn(),
            dispatch: jest.fn(),
            workouts: [{id: 1}],
        }
        wrapper = shallow(
                    <Provider store={mockProps}>
                        <WorkoutsLoggedPageContainer {...mockProps} />
                    </Provider>)
        expect(wrapper).toMatchSnapshot();
    })
    it("expect to render workout page", () => {
        const mockProps = {
            workouts: [{date: new Date(), id: 1}],
        }
        wrapper = shallow(<WorkoutsLoggedPage {...mockProps} />)
        expect(wrapper).toMatchSnapshot();
    })
});