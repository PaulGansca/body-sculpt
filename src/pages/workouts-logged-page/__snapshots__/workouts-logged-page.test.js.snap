// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkoutsLoggedPage /> container renders workout based on id 1`] = `ShallowWrapper {}`;

exports[`<WorkoutsLoggedPage /> expect to render workout page 1`] = `ShallowWrapper {}`;
